test-kit/components/tree-view-driver.ts

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import {DriverBase, simulate} from 'test-drive-react';
+import {TreeItem, TreeItemProps, TreeView} from '../../src';
+
+const treeView = 'TREE_VIEW';
+const treeItem = 'TREE_ITEM';
+
+const getTreeItem = (id: string) => `${treeItem}_${id.replace(' ', '_')}`;
+
+export class TreeViewDriver extends DriverBase {
+    public static ComponentClass = TreeView;
+
+    public getItemDriver(id: string): TreeItemDriver {
+        return new TreeItemDriver(() => this.select(getTreeItem(id)), id);
+    }
+
+    public toggleItem(id: string): void {
+        simulate.click(this.getItemDriver(id).icon);
+    }
+
+    public selectItem(id: string): void {
+        simulate.click(this.getItemDriver(id).label);
+    }
+
+    public pressKey(keyCode: number): void {
+        simulate.keyDown(this.select(treeView), {keyCode});
+    }
+}
+
+export class TreeViewInstanceDriver extends DriverBase {
```

Comment:
This driver seems somewhat redundant. The use case is also taking the `result` from the render, creating the driver, and then using it like you would use the result (give or take the getter functions).
Trying to consider a solution where you could use the regular TV driver to invoke the API (which is subject to change anyway) @tobich